app/controllers/api_controller.rb

```diff
@@ -0,0 +1,15 @@
+class ApiController < ApplicationController
+ 
+
+def index
+  	@plays = Theater.all
+  	render :json => 
+  		@plays.to_json(:only => [:name], 
+  						:include => {
+  						  :plays => {:only => [:title]}
+  						  }
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.